app-api/src/main/resources/liquibase-sender.xml

```diff
@@ -250,5 +250,74 @@
             </column>
         </addColumn>
     </changeSet>
+    
+    <changeSet author="lpinto" id="20220210-1050">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="sender_sync_message" />
+            </not>
+        </preConditions>
+        <comment>Adding sender_sync_message table</comment>
 
+        <createTable tableName="sender_sync_message">
+            <column autoIncrement="true" name="id" type="BIGINT">
+                <constraints nullable="false" primaryKey="true" />
+            </column>
+            <column name="table_name" type="VARCHAR(100)">
+                <constraints nullable="false" />
+            </column>
+            <column name="identifier" type="VARCHAR(255)">
+                <constraints nullable="false" />
+            </column>
+            <column name="operation" type="VARCHAR(1)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="message_uuid" type="VARCHAR(38)">
+                <constraints nullable="false" unique="true"/>
+            </column>
+            <column name="request_uuid" type="VARCHAR(38)">
+                <constraints unique="true"/>
```

Comment:
Remove unique constraint

Reply:
This is the sender_sync_message table. There is no chance to have duplicates here, since the request_uuid is generated per request.
I think that you wanna tell to remove in the sync_response table, where we could have multiple responses for the same request_uuid. Right?

Reply:
Recall that the receiver can request an entity from the a remote site, usually that request uuid remains is what gets copied over to this column, recall that? So it's not true that the sender always generates a new one

Reply:
Sorry, I'd misunderstood. I was switching concepts between **request_uuid** and **message_uuid**.
Fixed.